docs/getting-started-step-by-step/00-overview.md

```diff
@@ -0,0 +1,44 @@
+# Overview Getting Started: Step-by-Step
+
+The following guide provides an easy, step-by-step path for developers and platform engineers to get familiar with Crossplane. It complements the current Getting Started guide, which focuses on getting an RDS Instance up and running quickly. In this example will use Object Storage as a simple example with quick turn around times.
+
+In this guide we are going to all important aspects of Crossplane: Managed Resources, Provider, ProviderConfigs, Compositions, CompositeResourceDefinitions (XRD), CompositeResources (XR), Claims and Configurations. 
+
+Here is the full architectural overview: 
+
+![architecture-overview](00-architecture-overview.jpg)
```

Comment:
It's possible that I'm just too deep in the weeds to appreciate this diagram, but it seems perhaps a little misleading to me. Are we really going for a "full architectural overview" here, or is this intentionally simplified?

Reply:
Personally, I'm a fan of the diagram since it helps explain the idea of composition at a really high level from the operator's perspective (seeing both the definitions they create as well as the claim objects created by developers). Giving new users a mental model with a visual right away will be helpful as they work their way through the rest of the docs.

Reply:
@grantgumina - I’m all for an approachable diagram! My point is that we should be clear that’s what this is, and iterate it to be a little more accurate with regard to how our concepts relate to each other.

Reply:
+1 I've changed "Here is the full architectural overview: " => "Here is an architectural overview: "

I've refrained from adding more explanation that this is a simplification as this would add more context for new users.  We could add a link at the bottom in "More Info".

Reply:
Thanks @luebken! I really like the style and straightforwardness of this diagram, but I'm struggling to reconcile it with my understanding of how Crossplane works. I wonder if we can find a better compromise between simple, accurate, and consistent with our existing terminology.

Some thoughts.

1. Are composite resources and claims intentionally conflated here? If so, where does the reader go to learn the differences between the two?
2. Can we use the established term "defines" to model the relationship between a SimpleBucket and its XRD? This is a much more accurate and consistent way to frame the relationship - the SimpleBucket does not use its XRD but rather exists because of it.
3. Is the API line really at the XRD layer, or at the SimpleBucket layer? This kind of goes back to my above question - I'd typically think of the XRD sitting off to the left or right of the SimpleBucket rather than being part of the app consumption flow.

Reply:
Speaking of the style - I'd love to update all of our diagrams to use this style consistently in a future PR.